Document how to update Agentless CloudFormation and Terraform versions (#26353)
DataDog/documentation: content/en/security/cloud_security_management/setup/agentless_scanning/quick_start.md

{{% csm-agentless-exclude-resources %}}

## Update the CloudFormation stack

> Datadog recommends updating the CloudFormation stack regularly, so you can get access to new features and bug fixes as they get released. To do so, follow these steps:
> 1. Log in to your AWS console and go to the CloudFormation Stacks page.
> 2. Select the `DatadogIntegration-DatadogAgentlessScanning-...` CloudFormation sub-stack, and click on `Update` then `Update nested stack`.
> 3. Click on `Replace existing template`.
> 4. In the following S3 URL: `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`, replace `<VERSION>` with the version found in [aws_quickstart/version.txt][11].
> 5. Paste the S3 URL and click on `Next` in the next pages, then submit the form.

Reviewer: ```suggestion
Datadog recommends updating the CloudFormation stack regularly, so you can get access to new features and bug fixes as they get released. To do so, follow these steps:
1. Log in to your AWS console and go to the CloudFormation Stacks page.
2. Select the **DatadogIntegration-DatadogAgentlessScanning-...** CloudFormation sub-stack, click **Update**, then click **Update nested stack**.
3. Click **Replace existing template**.
4. In the following S3 URL: `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`, replace `<VERSION>` with the version found in [aws_quickstart/version.txt][11]. Paste that URL.
5. Click **Next**, then submit the form.
```

To uninstall Agentless Scanning, log in to your AWS console and delete the CloudFormation stack created for Agentless Scanning.

## Update the CloudFormation template version

> To update the Agentless Scanning CloudFormation template:

Reviewer 2: same 
```suggestion
To update the Agentless Scanning CloudFormation stack:
```

* Log in to your AWS console, select the `DatadogIntegration-DatadogAgentlessScanning...` CloudFormation sub-stack and click on `Update nested stack`
* Replace existing template using the following S3 URL:
  * `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`

>   * Replace `<VERSION>` in the URL above with the version found in https://github.com/DataDog/cloudformation-template/blob/master/aws_quickstart/version.txt

Author: Done.

To uninstall Agentless Scanning, log in to your AWS console and delete the CloudFormation stack created for Agentless Scanning.

## Update the CloudFormation template version

To update the Agentless Scanning CloudFormation template:

* Log in to your AWS console, select the `DatadogIntegration-DatadogAgentlessScanning...` sub-stack and click on `Update nested stack`
* Replace existing template using the following S3 URL:

>   * https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/v2.0.2/datadog_agentless_scanning.yaml
>   * The `v2.0.2` version in the URL above should be updated using the latest version from https://github.com/DataDog/cloudformation-template/blob/master/aws_quickstart/version.txt

Reviewer 3: ```suggestion
  * `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`
  * Replace `<VERSION>` in the URL above with the version found in https://github.com/DataDog/cloudformation-template/blob/master/aws_quickstart/version.txt
```
